pywr/recorders/_recorders.pyx

```diff
@@ -292,6 +292,7 @@ cdef class AggregatedRecorder(Recorder):
         recorder_names = data.pop("recorders")
         recorders = [load_recorder(model, name) for name in recorder_names]
         rec = cls(model, recorders, **data)
+        return rec
```

Comment:
Good spot. Type annotations / linting might have caught this.

We could also (in another PR?) check if the load methods always return a `Component`?

Reply:
I'll move this to a separate issue.

Reply:
See #727